Code/Common/include/sitkTransform.h

```diff
@@ -93,7 +93,7 @@ class SITKCommon_EXPORT Transform
    */
   template<unsigned int NDimension>
   explicit Transform( itk::CompositeTransform< double, NDimension >* compositeTransform )
-    : m_PimpleTransform( nullptr )
+  : Transform()
```

Comment:
The original line of code `: m_PimpleTransform( nullptr )` worked well already, right?

Reply:
This where the compilation error I encountered before occoured. Since this in in the header, the std::unique_ptr can not be initialized/assigned/constructed without the Pimple being defined. Hence by calling the compiled default constructor from cxx, the definition of the Pimple is not needed.

Reply:
ah, that's clever 👍